static/src/javascripts/projects/common/modules/media/videojs-plugins/skip-ad.js

```diff
@@ -0,0 +1,70 @@
+define([
+    'qwery',
+    'bean',
+    'common/utils/$',
+    'common/utils/template',
+    'text!common/views/ui/video-ads-skip-overlay.html'
+], function(
+    qwery,
+    bean,
+    $,
+    template,
+    adsSkipOverlayTemplate
+) {
+
+    function skipAd(mediaType, skipTimeout) {
+        var intervalId;
+        var events = {
+                update: function () {
+                    var adsManager  = this.ima.getAdsManager(),
+                        currentTime = adsManager.getCurrentAd().getDuration() - adsManager.getRemainingTime(),
+                        skipTime    = parseInt((skipTimeout - currentTime).toFixed(), 10);
+
+                    if (skipTime > 0) {
+                        $('.js-skip-remaining-time', this.el()).text(skipTime);
+                    } else {
+                        window.clearInterval(intervalId);
+                        $('.js-ads-skip', this.el())
+                            .html(
+                                '<button class="js-ads-skip-button vjs-ads-skip__button" data-link-name="Skip video advert">' +
```

Comment:
Trying to touch as little as possible - and this is all going to go away soon anyhow - so not worth the effort.

The point of this is to help clear out the events file.